editor.planx.uk/src/pages/FlowEditor/lib/store/user.ts

```diff
@@ -1,20 +1,14 @@
-import { User, UserTeams } from "@opensystemslab/planx-core/types";
-import gql from "graphql-tag";
+import { User } from "@opensystemslab/planx-core/types";
+import { _client } from "client";
 import jwtDecode from "jwt-decode";
-import { client } from "lib/graphql";
 import { Team } from "types";
 import type { StateCreator } from "zustand";
 
 export interface UserStore {
-  id: number;
-  firstName: string;
-  lastName: string;
-  email: string;
-  isPlatformAdmin: boolean;
-  teams: UserTeams[];
+  user?: User
```

Comment:
I consolidated these into a single user because of a side effect of merging our Zustand stores I was hitting up against.

FlowId is also called `id` in the stores and then overwrites userId which we wouldn't expect.
![image](https://github.com/theopensystemslab/planx-new/assets/20502206/0c1f032b-fe3d-4289-a9f7-b9193c665013)